collectors/feature/jira/src/main/java/com/capitalone/dashboard/client/team/TeamDataClientImpl.java

```diff
@@ -63,13 +60,6 @@ public int updateTeamInformation() {
 		if (CollectionUtils.isNotEmpty(teams)) {
 			updateMongoInfo(teams);
 			count += teams.size();
-		} else {
-			List<BasicProject> projects = jiraClient.getProjects();
-			
-			if (CollectionUtils.isNotEmpty(projects)) {
-				updateMongoInfoLegacy(projects);
-				count += projects.size();
-			}
```

Comment:
This is what I am thinking in terms of making it uniform across VersionOne and Jira and also matching the overall design pattern of rest of Hygieia.

1. Do not create any collector item from either VersionOne or Jira collector. Only create team and/or scope documents. 
2. UI drop downs fetches data from scope (for Project drop down) and team (for team drop down)
3. Once UI config is saved, create a ScopeOwnerCollectorItem with the combination configured. The collector item will have the project and team selected on the UI. One of these has to be present, both can't be null (error validation on UI)
4. Jira collector and VersionOne collector collects feature data based on the project+team in the collector item collection. This can potentially save a lot of time in data collection too.